config-service/build.gradle.kts

```diff
@@ -80,6 +80,9 @@ dependencies {
     runtimeOnly("io.netty:netty-handler-proxy:4.1.59.Final") {
       because("https://snyk.io/vuln/SNYK-JAVA-IONETTY-1070799")
     }
+    runtimeOnly("org.eclipse.jetty:jetty-server:9.4.37.v20210219") {
+      because("https://snyk.io/vuln/SNYK-JAVA-ORGECLIPSEJETTY-1080611")
```

Comment:
jetty is from platform-service-framework - upgrading that to latest would be more appropriate than constraining a version here - we only want to use constraints when the vulnerability is coming via a third party we can't update.

Reply:
Makes sense. Upgraded platform-service-framework version and removed this constraint